src/EnergyPlus/Autosizing/Base.cc

```diff
@@ -236,7 +236,7 @@ void BaseSizer::preSize(EnergyPlusData &state, Real64 const _originalValue)
                 }
             }
         }
-        if (this->unitarySysEqSizing.allocated())
+        if (allocated(this->unitarySysEqSizing))
```

Comment:
I take it this change allows EPVector to stay slimmed down by not needing a member method for `allocated`?  Then instead just include a header to access a utility function for it when needed?  Or something?